packages/taiko-client/bindings/encoding/input.go

```diff
@@ -591,8 +591,8 @@ func CalculatePacayaDifficulty(blockNum *big.Int) ([]byte, error) {
 	return packed, nil
 }
 
-// UnpackTxListBytes unpacks the input data of a TaikoL1.proposeBlock transaction, and returns the txList bytes.
-func UnpackTxListBytes(txData []byte) ([]byte, error) {
+// UnpackOntakeTxListBytes unpacks the input data of a TaikoL1.proposeBlock transaction, and returns the txList bytes.
+func UnpackOntakeTxListBytes(txData []byte) ([]byte, error) {
```

Comment:
After think twice, maybe we won't this method anymore? since we switched to fetch the txList from the event data?

Reply:
Yeah, later this function won't be used anymore.